### ISSUE TITLE
How to find the package that required a package that caused a conflict?

### ISSUE THREAD

**Issue by Silnar** [rmarquis/pacaur#517]
Hi, I'm trying to update AUR packages, but I've encountered the following problem:

```
LANG=C pacaur -Sau                                                                                                              ⏎
:: Starting AUR upgrade...
:: resolving dependencies...
:: looking for inter-conflicts...
:: mesa-libgl and catalyst-libgl are in conflict (libgl). Remove catalyst-libgl? [y/N] n
:: unresolvable package conflicts detected
:: failed to prepare transaction (conflicting dependencies)
:: mesa-libgl and catalyst-libgl are in conflict
```

Could you tell me how can I find the AUR package that requires mesa-libl ?

**Comment by rmarquis** [comment]
You guys are killing me. See #515.

**Comment by Silnar** [comment]
Hopefully not :smile: 

I think that you did't understood me correctly, so here you have a better explanation.

I encounter this problem twice a year and never know how to solve it quickly.

Ideally I would like to see message like
```
:: mesa-libgl (required by ??) and catalyst-libgl are in conflict (libgl). Remove catalyst-libgl? [y/N] n
```

where `??` is the name of AUR package that pacaur is trying to update.
It would be helpful to know the name of this package, so I can remove it for example.

It's easy to get a list of packages that requires `catalyst-libgl` with `pacman -Sii package-name`:

```
LANG=C pacman -Sii catalyst-libgl
Repository      : catalyst-hd234k
Name            : catalyst-libgl
Version         : 13.1-8
Description     : AMD/ATI Catalyst legacy drivers. Catalyst drivers libraries symlinks.
Architecture    : x86_64
URL             : http://www.amd.com
Licenses        : custom
Groups          : None
Provides        : libgl
Depends On      : catalyst-utils  mesa>=10.1.0-4
Optional Deps   : None
Required By     : 0ad  abuse  alienarena  allegro  amdstream  apitrace  assaultcube  blobby2  btanks  cairo  caph  clanlib  clementine
                  coin  compton  critter  dangerdeep  dgen-sdl  directfb  dosbox  epplet-base  erlang-sdl  freedroidrpg  freeglut
                  gl2ps  glee  glfw  glfw-doc  glhack  glitz  glob2  glu  gnome-session  gnustep-back  haskell-openglraw  higan
                  irrlicht  irrlicht-docs  kmscon  libasl  libreoffice-fresh  libreoffice-still  libretro-mupen64plus  libretro-reicast
                  libva  libva-vdpau-driver  libvdpau-va-gl  lincity-ng  love08  luxmark  luxrays  luxrender  megaglest  mesa-demos
                  mplayer  mpv  mysql-workbench  mythtv  naev  neverball  nvidia  nvidia-304xx  nvidia-304xx-lts  nvidia-340xx
                  nvidia-340xx-lts  nvidia-dkms  nvidia-lts  opencsg  openra  pingus  processing  projectm  projectm-qt  projectm-test
                  pymol  qemu  qt4  qt5-base  retroarch  sauerbraten  scummvm  sdl2  simplescreenrecorder  squeak-vm  stellarium
                  tigervnc  tiled  ufoai  uqm  webkitgtk  webkitgtk2  wlc  xawtv  xorg-server  xorg-server-xephyr  xorg-server-xvfb
                  xorg-server-xwayland  xorg-xdriinfo
Optional For    : alsaplayer  catalyst-utils
Conflicts With  : libgl
Replaces        : None
Download Size   : 1456.00 B
Installed Size  : 37888.00 B
Packager        : Vi0L0 <vi0l093@gmail.com>
Build Date      : Sat Mar 15 17:13:23 2014
MD5 Sum         : 64da47185f8058811c65a40666345895
SHA-256 Sum     : 92721476a19e4ca9cc16b943c9c47e899cdab37a3f16f7a8fc8d2e1906943886
Signatures      : ABED422D653C3094
```

but searching for AUR packages in similar fashion using pacaur doesn't return anything:

```
LANG=C pacaur -Saii mesa-libgl
:: no results found
```

To sum up:

- I'm updating aur packages
- I know that at least one of them requires mesa-libgl and causes conflicts
- I don't know which one

Cheers.

**Comment by rmarquis** [comment]
In that case your request in similar to #173. And there is no easy way to achieve this, as the RPC interface is quite limited.

**Comment by Silnar** [comment]
Ok. I see. Thanks for info.
